### ISSUE TITLE
Enunciate and using my own spring bean for Service

### ISSUE THREAD

**Issue by jmjava** [stoicflame/enunciate#88]
Is it possible to not have enunciate generate the service bean and to use one from my spring bean config?  I can't figure out how to inject dependencies into the bean created with the enunciate Factory.

When i use spring Import it will load my beans that i have configured in other files great... but the service bean only seems to support handlers and interceptors... what if i want to delegate from that bean into another bean?  i can't seem to find a way to inject my bean into the bean generated by enunciate.

thanks for any help.  this is a great project

**Comment by stoicflame** [comment]
The factory will first look to see if there's already a bean defined in your context of that type. If so, it will use it.

It should just be a matter of defining the bean in your spring context. Have you tried it?

**Comment by jmjava** [comment]
Ryan,  thanks for the quick reply.  I tried and it was still generating the
bean in applicationContext.xml

Let me try again maybe i had something out of place.

Is coding all the CXF endpoints myself an option ?  Would i be missing out
on something?  I like the other apis that enunciate will generate.

CXF endpoints are easy enough to create just wondering about the other
listeners that are created in web.xml. Would i be missing out on some
Enunciate features???

thanks again for the reply.

John

**Comment by stoicflame** [comment]
> I tried and it was still generating the bean in applicationContext.xml

The beans that get generated _should_ be instances of `org.codehaus.enunciate.modules.spring_app.ServiceEndpointFactoryBean`, which use the beans you declared yourself.

> Is coding all the CXF endpoints myself an option ? 

Sure.

> Would i be missing out on something?

I can't think of anything.
